packages/design-system-shared/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@metamask/design-system-shared",
   "version": "0.0.0",
-  "private": true,
```

Comment:
Originally I thought we could make this package private but we need to publish it to the npm registry for our other packages to use it